src/gpo/assets/en-US/PowerToys.adml

```diff
@@ -27,6 +28,12 @@ If you enable this setting, the utility will be always enabled and the user won'
 If you disable this setting, the utility will be always disabled and the user won't be able to enable it.
 
 If you don't configure this setting, users are able to disable or enable the utility.
+</string>
+      <string id="AllowExperimentationDescription">This policy configures whether PowerToys experimentation is allowed. With experimentation allowed the user sees the new features being experimented if it gets selected as part of the test group. (Experimentation will only happen on Windows Insider builds.)
+
+If this setting is not configured or enabled, experimentation is allowed.
```

Comment:
```suggestion
If this setting is not configured or enabled, the user can control experimentation in the PowerToys settings.
```

To be correct about what happens.

Reply:
[Off topic] Do we have the same incorrectness on the other policies too?

Reply:
I don't think we do. (Or at least it's sort of implied that it can be controlled in Settings)

Reply:
It's only if it's selected, I think 🤷 Might be sort of confusing.

Reply:
Edge and Windows does it the same. There are GPOs that say: If xxx the user is in control.